torch/csrc/inductor/aoti_model_runner.cpp

```diff
@@ -0,0 +1,59 @@
+#if !defined(C10_MOBILE) && !defined(ANDROID)
+#include <ATen/DynamicLibrary.h>
+
+#include <torch/csrc/inductor/aoti_model_runner.h>
+#include <torch/csrc/inductor/aoti_torch/tensor_converter.h>
+
+namespace torch::inductor {
+
+AOTIModelRunner::AOTIModelRunner(
```

Comment:
Can we use some refactoring to share the code between AOTIModelRunner and AOTIModelContainerRunner?

Reply:
I don't think it's that trivial after some look. It does seem like they are sharing lots of things, but in fact the types are all different, and since we're also loading from .so and using decltype for variables, it's not that easy to share/templatize.